src/Akka.Streams.Kafka/Helpers/PartitionEventHandler.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Immutable;
+using Akka.Streams.Stage;
+using Confluent.Kafka;
+
+namespace Akka.Streams.Kafka.Helpers
+{
+    /// <summary>
+    /// This interface is used to pass callbacks when Kafka rebalances partitions between consumers,
+    /// or an kafka consumer is stopped
+    /// </summary>
+    internal interface IPartitionEventHandler
```

Comment:
This interface is public in alpakka implementation, but making it public will require passing it's implementations as an arguments in different places, adding more checks etc - so left it internal for now.
It is already used by stages to provide their callback for partition events, but making it public in future will allow library users to subscribe with their own partition handlers (or kind of this).
Also worth to mention that this interface is marked with `@ApiMayChanged` attribute in scala, so maybe even better to keep it internal for now.

Reply:
You should be able to use the `ApiMayChange` attribute from Akka - that's implemented in the project:

https://github.com/akkadotnet/akka.net/blob/41c116d2a3d280481e584401608ec848dc147632/src/core/Akka/Annotations/Attributes.cs - go ahead and add that where it's needed.

Reply:
See my other comments about `ApiMayChange` in Akka.NET